dotcom-rendering/src/web/components/SignInGateSelector.importable.tsx

```diff
@@ -207,13 +237,28 @@ export const SignInGateSelector = ({
 		return null;
 	}
 
-	const componentId = signInGateTestIdToComponentId[currentTest.id];
+	const personalisedComponentId = (
```

Comment:
Just wondering if worth moving this personalisation code block and the related state hook to the comment block above so everything is super co-located in the file?

Reply:
Yep can move that above